content/200-concepts/100-components/01-prisma-schema/03-generators.mdx

```diff
@@ -76,3 +76,4 @@ The following is a list of community created generators.
 - [`prisma-class-generator`](https://github.com/kimjbstar/prisma-class-generator): Generates classes from your Prisma Schema that can be used as DTO, Swagger Response, TypeGraphQL and so on.
 - [`zod-prisma`](https://github.com/CarterGrimmeisen/zod-prisma): Creates Zod schemas from your Prisma models.
 - [`prisma-giraphql-types`](https://github.com/hayes/giraphql/tree/main/packages/plugin-prisma): Makes it easier to define Prisma-based object types, and helps solve n+1 queries for relations. It also has integrations for the Relay plugin to make defining nodes and connections easy and efficient.
+- [`graphql-schema-generator`](https://github.com/prisma-korea/graphql-schema-generator): Generates GraphQL schema from Prisma schema without [GraphQL Nexus](https://nexusjs.org/docs/).
```

Comment:
Maybe do not describe via what it does _not_ do (without Nexus) but describe that it creates a plain GraphQL schema for any GraphQL tool, schema first I assume?

Reply:
Ok. I agree. I updated it.